doctr/utils/geometry.py

```diff
@@ -213,14 +213,18 @@ def rotate_boxes(
 def rotate_image(
     image: np.ndarray,
     angle: float,
-    expand=False,
+    expand: bool = False,
+    keep_original_size: bool = False,
```

Comment:
Don't you think this can be done by adding a resize transformation after the rotation instead?